hbase-server/src/main/java/org/apache/hadoop/hbase/master/RegionStates.java

```diff
@@ -824,6 +824,11 @@ public void regionOffline(
     }
 
     for (HRegionInfo hri : regionsToOffline) {
```

Comment:
I saw your description in https://issues.apache.org/jira/browse/HBASE-26518, it says this issue could be reproduced. So we should also write a UT about it?